### PR TITLE
Newsletter-137:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

**Review by bitschmidty — COMMENTED** [review]
@azuchi Thanks so much for these translations efforts ongoing!

I was able to build and check all the links

File: _posts/ja/newsletters/2021-02-24-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  `btcwire 0.5.0/neutrino`というユーザーエージェントから多数の接続（75%）と帯域幅使用量（90%）があることに気付きます。
+  Murchは、これらのピアはLNDノードであり、Compact Block FilterはBitcoin Coreの[新機能][news132 0.21.0]で、
+  かつ[デフォルトでは無効になっている][news98 18877]ため、現在ネットワーク上でピアにCompact Block Filterを提供するノードが不足しており、
+  その結果、それをサポートするノードへのトラフィックが高くなっている可能性があります。
+
+- [<!--is-there-dumpwallet-output-documentation-explanation-->`dumpwallet`の出力に関するドキュメントや説明はありますか？]({{bse}}101767)
+  Andrew Chowは、この機会に、`dumpwallet`RPCの出力の説明を回答しています。
+
+- [<!--is-there-something-about-bitcoin-that-prevents-implementing-the-same-privacy-protocols-of-monero-and-zcash-->BitcoinにMoneroやZcashと同様のプライバシープロトコルを実装することを妨げるものはありますか？]({{bse}}101868)
+  Pieter Wuilleは、MoneroやZcasが採用しているプライバシー強化のアプローチについて、開発者が取り組むことを選択しない理由や、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-24-newsletter.md:62]: s/Zcas/Zcash/

```suggestion
  Pieter Wuilleは、MoneroやZcashが採用しているプライバシー強化のアプローチについて、開発者が取り組むことを選択しない理由や、
```

File: _posts/ja/newsletters/2021-02-24-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- [Bitcoin Core #19136][]は、アドレスの公開鍵を含むウォレットの
+  [output script descriptor][topic descriptors]を含む新しい`parent_desc`フィールドで
+  `getaddressinfo` RPCを拡張しています。ウォレットの[BIP32][]パスは、
+  すべての強化導出プレフィックスが除去され、公開鍵の導出手順のみが残ります。
+  これにより、アドレスとその兄弟アドレスが受信したビットコインを監視可能な他のウォレットソフトウェアに
+  descriptorをインポートすることができます。
+
+- [Bitcoin Core #15946][]では、`prune`と`blockfilterindex`の両方の設定オプションを同時に使用して、
+  プルーニングされたノードで[Compact Block Filter][topic compact block filters]を保持することが可能になりました
+  （`peerblockfilters`設定オプションが使用されている場合は、それらも提供されます）。LNDの開発者は、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-24-newsletter.md:95]: s/それらも提供されます/フィルターの提供もされます/

```suggestion
  （`peerblockfilters`設定オプションが使用されている場合は、フィルターの提供もされます）。LNDの開発者は、
```

File: _posts/ja/newsletters/2021-02-24-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  これは自社のソフトウェアにとって有益で、将来のアップデートではウォレットロジックがBlock Filterを使用してウォレットをインポートするために、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-24-newsletter.md:96]:
```suggestion
  これは自社のソフトウェアにとって有益で、将来のアップデートではウォレットをインポートするために、ウォレットロジックがBlock Filterを使用して
```

File: _posts/ja/newsletters/2021-02-24-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+数少ない情報ソースです。この月刊セクションでは、前回アップデート以降にされた、最も票を集めた質問・答えを紹介しています。*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [<!--is-sharding-a-good-alternative-to-multisig-->シャーディングはマルチシグの良い代替手段ですか？]({{bse}}102007)
+  ユーザーS.O.Sは、シャミアの秘密分散（SSS）のようなシャーディング方式でマルチシグのような機能を実現することの可能性について質問しています。
+  Pieter Wuilleは、SSSに対する`OP_CHECKMULTISIG`の利点（アカウンタビリティ、1台のマシン上で秘密鍵を再構築する必要がない）、
+  `OP_CHECKMULTISIG`に対するSSSの利点（手数料が少なく、プライバシーの強化）および、
+  [Schnorr][topic schnorr signatures]の考慮事項（アカウンタビリティは欠如するが、1台のマシンで秘密鍵を再構築する必要はなく、
```
[the remark this answers is not in the document]
Comment (azuchi) [reply _posts/ja/newsletters/2021-02-24-newsletter.md]: `また、` is ommited because previous line end with `および、`

**Comment by azuchi** [comment]
@AkioNak Thanks! I fixed.

File: _posts/ja/newsletters/2021-02-24-newsletter.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (AkioNak) [reply _posts/ja/newsletters/2021-02-24-newsletter.md]: Oh, I missed that.

File: _posts/ja/newsletters/2021-02-24-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
+---
+title: 'Bitcoin Optech Newsletter #137'
+permalink: /ja/newsletters/2021/02/24/
+name: 2021-02-24-newsletter-ja
+slug: 2021-02-24-newsletter-ja
+type: newsletter
+layout: newsletter
+lang: ja
+---
+今週のニュースレーターでは、Taprootのソフトフォークのアクティベーションパラメーターの選択に関する議論の結果を説明と、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-24-newsletter.md:10]: @azuchi I am sorry for my incomplete suggestion.
"議論の結果の説明と" is better.

```suggestion
今週のニュースレーターでは、Taprootのソフトフォークのアクティベーションパラメーターの選択に関する議論の結果の説明と、
```

Comment (azuchi) [reply _posts/ja/newsletters/2021-02-24-newsletter.md]: Oops! Sorry!

**Comment by AkioNak** [comment]
@azuchi Thanks! It looks good to me.

**Comment by bitschmidty** [comment]
Merged! Thanks @azuchi and @AkioNak !

**Pull request merged**